timemachine/maps/estimators.py

```diff
@@ -0,0 +1,56 @@
+"""Utilities for computing mapped works or mapped u_kn matrix, incorporating invertible map_fxns"""
+
+import numpy as np
+from pymbar.utils import kln_to_kn  # TODO: should this be "vendored"?
```

Comment:
Currently importing from https://github.com/choderalab/pymbar/blob/1d14e80d3320a5ae0651fd002a8cb763a900190c/pymbar/utils.py#L46-L85 , but could be replicated here

Reply:
`kln_to_kn(u_kln)` seems like a synonym for `np.concatenate(u_kln, axis=1)`? The latter seems to be ~10x faster according to some rough timing. Is there any reason to prefer the former?

Reply:
>kln_to_kn(u_kln) seems like a synonym for np.concatenate(u_kln, axis=1)?

Not quite, in the general case that N_k are not all equal.

>The latter seems to be ~10x faster according to some rough timing. Is there any reason to prefer the former?

Other simplifications can be applied relative to the `kln_to_kn` utility in pymbar (e.g. assume `N_k` is always given / doesn't have to be inferred) -- just used this initially without peaking inside the definition.